packages/main/src/Carousel.js

```diff
@@ -110,6 +111,23 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
```

Comment:
Please only change line 103 to:
* Defines the visibility of the page indicator.